mobile_packet_verifier/pkg/settings-template.toml

```diff
@@ -10,6 +10,10 @@ cache = "/var/data/verified-reports"
 # Default value is 1 hour.
 burn_period = 1
 
+# In case burn fails we will accelarate and burn in `min_burn_period` minutes instead of `burn_period` hours.
```

Comment:
```suggestion
# In case burn fails we will accelerate and burn in `min_burn_period` minutes instead of `burn_period` hours.
```

It feels a bit weird to have 2 different units of time used this close together and only mentioned in the comments. I'm not sure how easy it would be to change both `burn_period_hrs` and `min_burn_period_mins`.